### PR TITLE
Add known vector index issue where queries could be incorrectly rejected

### PR DESCRIPTION
**Pull request by nilsceberg**

### CODE REVIEW

**Comment by nilsceberg** [comment]
[[S2][Pfizer - Manufacturing Excellence Solutions][Neo4j 5.13] Index query not supported for VECTOR index](https://trello.com/c/2rlB5KIM/9353-s2pfizer-manufacturing-excellence-solutionsneo4j-513-index-query-not-supported-for-vector-index)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/indexes-for-vector-search.adoc [hunk 1]
```diff
@@ -428,6 +428,14 @@ WHERE type = 'VECTOR'
 ====
 | Neo4j 5.15
 
+| Vector indexes may incorrectly reject valid queries in a cluster setting. This is caused by a bug in the handling of index capabilities on followers.
+
+[TIP]
+====
+Because index capabilities will be correctly configured on a restart, this issue can be worked around by rolling the cluster after vector index creation.
+====
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/indexes-for-vector-search.adoc:490]:
```suggestion
====

For more information about clustering in Neo4j, see the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/clustering[Operations Manual -> Clustering].
```

File: modules/ROOT/pages/indexes-for-vector-search.adoc [hunk 2]
```diff
@@ -428,6 +428,14 @@ WHERE type = 'VECTOR'
 ====
 | Neo4j 5.15
 
+| Vector indexes may incorrectly reject valid queries in a cluster setting. This is caused by a bug in the handling of index capabilities on followers.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/indexes-for-vector-search.adoc]:
```suggestion
| Vector indexes may incorrectly reject valid queries in a cluster setting.
This is caused by an issue in the handling of index capabilities on followers.
```

**Review by JPryce-Aklundh — APPROVED** [review]
Looks good - just 2 small suggestions

**Pull request merged**